src/Template/Template.js

```diff
@@ -32,6 +34,28 @@ if (typeof HTMLTemplateElement === 'undefined') {
       while (child = template.firstChild) {
         template.content.appendChild(child);
       }
+      // add innerHTML to template
+      Object.defineProperty(template, 'innerHTML', {
+        get: function() {
+          var o = '';
+          for (var e = this.content.firstChild; e; e = e.nextSibling) {
+            if (e) {
+              o += e.outerHTML || e.textContent;
```

Comment:
Ah, found a better example: ShadowDOM's innerHTML, from this very repository :)
https://github.com/webcomponents/webcomponentsjs/blob/938051272519053867b5bafa7411e7afefdd2c01/src/ShadowDOM/wrappers/HTMLElement.js#L34